docs/finance.md

```diff
@@ -30,11 +30,17 @@ pageClass: routes
 
 ### 电报
 
-<Route author="nczitzk" example="/cls/telegraph" path="/cls/telegraph"/>
+<Route author="nczitzk" example="/cls/telegraph" path="/cls/telegraph/:category?" :paramsDesc="['分类，见下表']">
```

Comment:
Please include all route added in the route block.

Reply:
👌